core/admin/mailu/translations/ja/LC_MESSAGES/messages.po

```diff
@@ -3,17 +3,21 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: POEditor.com\n"
+"X-Generator: Poedit 1.5.7\n"
 "Project-Id-Version: Mailu\n"
 "Language: ja\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
```

Comment:
Ah! It seems these 2 either need to be a valid date, or be removed! Didn’t know myself … m(_ _)m